dbos/admin_server.go

```diff
@@ -226,7 +227,7 @@ func newAdminServer(ctx *dbosContext, port int) *adminServer {
 	mux.HandleFunc(_DEACTIVATE_PATTERN, func(w http.ResponseWriter, r *http.Request) {
 		if as.isDeactivated.CompareAndSwap(0, 1) {
 			ctx.logger.Info("Deactivating DBOS executor", "executor_id", ctx.executorID, "app_version", ctx.applicationVersion)
-			// TODO: Stop queue runner, workflow scheduler, etc
+			ctx.Cancel(1 * time.Minute) // Cancel context, queue runner, and workflow scheduler
```

Comment:
This completely breaks queue draining on old versions

Reply:
Good point, Cancel shouldn't stop the queue runner, just the workflow scheduler.